kernel/static/default-profile/download_all.js

```diff
@@ -19,7 +21,7 @@ const downloadFile = async (url, path) => {
   })
 }
 
-const catalog = require('./basecatalog.json')
+const catalog = WSS_ENABLED ? require('./basecatalog-wss.json') : require('./basecatalog.json')
```

Comment:
Shouldn't must we also do something like that inside the profile saga for this to work?
`const response = yield fetch(getResourcesURL() + '/default-profile/basecatalog.json')`